.github/workflows/qa.yml

```
@@ -9,6 +9,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@master
+      - name: Delete composer files to avoid installing dependencies
+        run: rm -v composer.json composer.lock
```

```suggestion
        run: rm -v composer.json
```

My bad, `composer.lock` is not versionned.